sdk/internal/recording/recording.go

```diff
@@ -85,340 +79,53 @@ const (
 	Secret_Base64String VariableType = "secret_base64String"
 )
 
-// defaultSanitizersSet tracks whether default sanitizers have been added (this would be neater with Go 1.19's atomic.Bool)
-var defaultSanitizersSet int32
+var (
+	// defaultSanitizersSet tracks whether default sanitizers have been added (this would be neater with Go 1.19's atomic.Bool)
+	defaultSanitizersSet int32
+
+	errUnsupportedAPI = errors.New("the vcr based recording API isn't supported. Use the test proxy instead")
```

Comment:
Maybe add the word "test" in here just in case, i.e. `...based test recording...`